duvet/src/text/find.rs

```diff
@@ -32,58 +52,214 @@ fn fast_find(needle: &str, haystack: &str) -> Option<Range<usize>> {
 
 /// TODO we should probaly deprecate this - it's better to enforce strict matching
```

Comment:
still believe this?

Reply:
yep. that's why i've added the ability to warn on it 😄. still need to wire up the warnings though.

Reply:
oh I read the TODO as suggesting not allowing fuzzy finds at all, so it wouldn't be a warning, it would just not match

Reply:
Yeah that's what I want to get to. But removing the fuzzy matching right now would be a breaking change so I'd like to give some buffer for codebases to fix the matches and then we can deprecate some time in the future.

Reply:
ah I get it, so the warning would be something like "this is a fuzzy match, which is deprecated and will no longer be supported"

Reply:
Yeah that's exactly what I had in mind